feature-libs/organization/account-summary/components/account-summary/details/document/filter/account-summary-document-filter.component.html

```diff
@@ -0,0 +1,98 @@
+      </ng-select>
+      <cx-form-errors [control]="filterForm.controls.status"></cx-form-errors>
+    </label>
+
+    <label class="cx-account-summary-document-filter-form-item">
+      <span>{{ 'orgAccountSummary.filter.filterBy' | cxTranslate }}</span>
+      <ng-select [searchable]="false" [clearable]="false" [items]="filterByOptions"
+                 bindLabel="name" bindValue="code" formControlName="filterBy">
+      </ng-select>
+      <cx-form-errors [control]="filterForm.controls.filterBy"></cx-form-errors>
+    </label>
+
+    <label class="cx-account-summary-document-filter-form-item" *ngIf="filterBy === FilterByOptions.DOCUMENT_TYPE">
+      <span>{{ 'orgAccountSummary.filter.documentType' | cxTranslate }}</span>
+      <ng-select [searchable]="false" [clearable]="false" [items]="documentTypeOptions"
+                 bindLabel="name" bindValue="code" formControlName="documentType">
+      </ng-select>
+      <cx-form-errors [control]="filterForm.controls.documentType"></cx-form-errors>
+    </label>
+
+    <label class="cx-account-summary-document-filter-form-item" *ngIf="filterBy === FilterByOptions.DOCUMENT_NUMBER">
+      <span>{{ 'orgAccountSummary.filter.documentNumber' | cxTranslate }}</span>
+      <input class="cx-account-summary-document-filter-form-input" formControlName="documentNumber"/>
+      <cx-form-errors [control]="filterForm.controls.documentNumber"></cx-form-errors>
+    </label>
+
+    <ng-template [ngIf]="filterBy === FilterByOptions.DOCUMENT_NUMBER_RANGE">
+      <ng-container *ngTemplateOutlet="rangeInput; context: { formGroup: filterForm.get('documentNumberRange'), type: 'text' }">
+      </ng-container>
+    </ng-template>
+
+    <ng-template [ngIf]="filterBy === FilterByOptions.DATE_RANGE">
+      <ng-container *ngTemplateOutlet="rangeInput; context: { formGroup: filterForm.get('documentDateRange'), type: 'date' }">
+      </ng-container>
+    </ng-template>
+
+    <ng-template [ngIf]="filterBy === FilterByOptions.DUE_DATE_RANGE">
+      <ng-container *ngTemplateOutlet="rangeInput; context: { formGroup: filterForm.get('dueDateRange'), type: 'date' }">
+      </ng-container>
+    </ng-template>
+
+    <ng-template [ngIf]="filterBy === FilterByOptions.AMOUNT_RANGE">
+      <ng-container *ngTemplateOutlet="rangeInput; context: { formGroup: filterForm.get('originalAmountRange'), type: 'number' }">
+      </ng-container>
+    </ng-template>
+
+    <ng-template [ngIf]="filterBy === FilterByOptions.OPEN_AMOUNT_RANGE">
+      <ng-container *ngTemplateOutlet="rangeInput; context: { formGroup: filterForm.get('openAmountRange'), type: 'number' }">
+      </ng-container>
+    </ng-template>
+
+    <div class="cx-account-summary-document-filter-form-button-block">
+      <button class="clear-btn" type="button" (click)="resetForm(true)">
+        {{ 'orgAccountSummary.filter.clear' | cxTranslate }}
+      </button>
+      <button class="search-btn" type="submit">
```

Comment:
primary buttons should have the `btn btn-primary`. With `search-btn` the button sytling is different from the rest of spartacus buttons.